py_analysis/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM socrata/python
+
+RUN apt-get update && \
+    apt-get install -y python-numpy \
+                       python-scipy
```

Comment:
apt-get commands should be prefixed with DEBIAN_FRONTEND=noninteractive to avoid cases where apt-get or various package scripts may want to prompt for input.

See base/Dockerfile for example.